packages/access-content/lib/mongo/index.mjs

```diff
@@ -0,0 +1,5 @@
+export * from './MongoRegisterAccess.mjs'
+export * from './MongoRepositoryAccess.mjs'
+export * from './MongoResourceAccess.mjs'
+export * from './MongoContentAccess.mjs'
+
```

Comment:
what is the difference between the register, repository, resource, and content?

Reply:
we need all these accesses just to get access to the repo?

Reply:
Repository here is not like `Repository` in DDD, I would say that whole package `access-content` it's a `DDD Repository`

`MongoResourceAccess` - already deprecated, in current version it's `MongoRegisterAccess` - should keep records of available repositories per  owner
`MongoRepositoryAccess` -  should provide a folder/file-like representation of content hierarchy , aka filesystem, keep permissions
`MongoContentAccess` - should keep context, provide base CRUD over content 
 

It's still messy, I'm planing to clean it up in next few days